R/R-QuickJSR/Portfile

```diff
@@ -6,24 +6,17 @@ PortGroup           R 1.0
 
 legacysupport.newest_darwin_requires_legacy 15
 
-R.setup             cran andrjohns QuickJSR 1.2.2
+R.setup             cran andrjohns QuickJSR 1.3.0
 revision            0
 categories-append   devel
 maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
 license             MIT
 description         Interface for the QuickJS light-weight JavaScript engine
 long_description    {*}${description}
 homepage            https://github.com/andrjohns/QuickJSR
-checksums           rmd160  b04456efdc5d047936f632dc820e255ca8258fab \
-                    sha256  51842f846585c8dfdab155eb5a26be0735cae280fa1c1bff1907d718525968ed \
-                    size    665175
-
-# https://github.com/andrjohns/QuickJSR/issues/55
-patchfiles          patch-right-compiler.diff
```

Comment:
Are we deleting this patch file if we're no longer using it?

Reply:
Should have been deleted. If not, I will fix it in a few minutes.

Reply:
Dropped the patch